src/pages/[...slug].tsx

```diff
@@ -42,14 +42,18 @@ interface Props {
 }
 
 export const getStaticPaths: GetStaticPaths = () => {
-  const contentFiles = getContent(["slug", "content"])
+  const contentFiles = getContent("/", ["slug"]).filter(
+    // Filter `/developers/tutorials` slugs since they are processed by
+    // `/developers/tutorials/[...tutorial].tsx`
+    (file) => !file.slug!.includes("/developers/tutorials")
+  )
 
   return {
     paths: contentFiles.map((file) => {
       return {
         params: {
           // Splitting nested paths to generate proper slug
-          slug: file.slug.split("/").slice(1),
+          slug: file.slug!.split("/").slice(1),
```

Comment:
can `slug` be undefined? otherwise `!` is not required

Reply:
This is related to the types of `getContentBySlug` where all the fields were marked as optional.

Reply:
Yeah I encountered this as well locally and had to add a bang